public/app.js

```diff
@@ -0,0 +1,330 @@
+// AUTH CONTROLLER
+var authController = (function() {
+    return {
+        //Authentication state listener
+        initApp: function() {
+            firebase.auth().onAuthStateChanged(function (user) {
+                if (user) {
+                    console.log('Login Successful');
+                    console.log("Attempting to bind: " + user.email)
+                    console.log('Bind Successful');
+                    return user;
+                } else {
+                    window.location.replace('auth.html')
+                }
+            });
+        },
+
+        //logout function
+        logout: function() {
+            firebase.auth().signOut()
+                .catch(function (err) {
+                    console.log(err);
```

Comment:
I think we should be much more noisy than just a console log. Maybe an alert() for now?

Reply:
Okay